monitor/cron_monitor.sh

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env bash
+# set -x
+# set -e
+
+export AWS_ACCESS_KEY_ID=$S3_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$S3_SECRET_ACCESS_KEY
+
+if [[ -z "$S3_ENDPOINT" ]]; then
+  AWS_ARGS=""
+else
+  AWS_ARGS="--endpoint-url $S3_ENDPOINT"
+fi
+
+list_s3() {
+  if [[ -z "$S3_PREFIX" ]]; then
+    S3_PATH= "s3://$S3_BUCKET"
+  else
+    S3_PATH="s3://$S3_BUCKET/$S3_PREFIX"
+  fi
+
+  if [[ ! -z "$1" ]]; then
+    S3_PATH="$S3_PATH/$1"
+  fi
+
+  echo "Listing all files from $S3_PATH"
+  aws $AWS_ARGS s3 ls $S3_PATH/ --human-readable
+}
+s3_latest_mediawiki_dump_backup_record=$(list_s3 hourly | tail -1) 
+echo "Latest mediawiki dump record: $s3_latest_mediawiki_dump_backup_record"
+s3_latest_sql_dump_backup_record=$(list_s3 hourly | tail -2 | head -1) 
```

Comment:
@vijayabharathib Instead of this can we search for `mediawiki.tar.gz` or `.dump.sql.gz` .. Becuase it won't detect if anyone of files is failed always.

Reply:
grepped it. done